shell/packages/sandstorm-db/db.js

```diff
@@ -1797,6 +1789,50 @@ _.extend(SandstormDb.prototype, {
     return setting && setting.value && this.isFeatureKeyValid();
   },
 
+  updateUserQuotaFromLdap: function (user) {
```

Comment:
I don't think this method belongs in `db.js`. It's specific to LDAP, it depends on `LDAP` being present, etc.

It seems like you're putting it here because `getUserQuota()` needs to call it, but do we really want to be pinging the LDAP server synchronously every time we need to look up a user's quota? If the LDAP server goes down temporarily, Sandstorm would be essentially inoperable.